FragmentService/FragmentServiceInterface.php

```diff
@@ -46,6 +46,15 @@ public function buildCreateForm(FormMapper $form, FragmentInterface $fragment);
     public function buildForm(FormMapper $form, FragmentInterface $fragment);
 
     /**
+     * Gets content to render fragment.
+     *
+     * @param FragmentInterface $fragment
```

Comment:
This is repetitive

Reply:
sorry but what do you mean by `This is repetitive`?

Reply:
It provides the exact same information the type hint already provides.

Reply:
ok so should i remove `@param FragmentInterface $fragment` ? sorry to ask but it seems not consistent with other method declarations/phpdoc?

Reply:
I'm not sure... WDYT @sonata-project/contributors ?